spikeinterface/toolkit/postprocessing/principal_component.py

```diff
@@ -354,7 +357,7 @@ def _fit_by_channel_local(self):
         # fit
         for unit_id in unit_ids:
             wfs = we.get_waveforms(unit_id)
-            if wfs.size == 0:
+            if len(wfs) < p['n_components']:
                 continue
```

Comment:
Should we raise an Exception/Warning as well? @samuelgarcia

Reply:
No I think continue is OK. We don't want to break the computation when we have to few spikes in one units.

Reply:
> No I think continue is OK. We don't want to break the computation when we have to few spikes in one units.

Ok then let's merge :) Thanks @miketrumpis